sycl/include/sycl/device_selector.hpp

```diff
@@ -87,5 +84,20 @@ class __SYCL_EXPORT host_selector : public device_selector {
 public:
   int operator()(const device &dev) const override;
 };
+
+namespace detail {
+
+// SYCL 1.2.1 defines a negative score to reject a device from selection
```

Comment:
Since it is the case for SYCL 2020 too
```suggestion
// SYCL 2020 section 4.6.1.1 defines a negative score to reject a device from selection
```